### DISCUSSION TITLE
https://gltf.pmnd.rs/ GTLF -> React-Three-Fibre working for you?

### DISCUSSION THREAD

**Discussion by handodk** [pmndrs/gltfjsx#233] [Q&A]
I'm fairly new to React Three Fibre. Following the docs/examples to try import my own 3D model.

Docs say this site will convert mt GTLF file into a re-usable React component: https://gltf.pmnd.rs/

I've tried to load a few different GTLF files but it seems to hang on "Loading" (dev console shows errors).

I've tried in different browsers. Etc.

Is it working for anyone else?

Thanks.

**Comment by nietoperq** [comment]
I have the same problem with .gltf files, but it works for .glb file extension, maybe try to convert your models to glb?

**Reply by kabir12505** [reply]
still not working
